app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert len(split_integer(8, 1)) == 1
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(6, 2) == [3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert len(split_integer(8, 1)) == 1
```

Comment:
The test `test_should_return_part_equals_to_value_when_split_into_one_part` is incorrectly asserting the length of the result. According to the task description, you should assert that the result is `[8]` when splitting into one part. Consider changing the assertion to `split_integer(8, 1) == [8]`.